users/tests/test_views.py

```diff
@@ -88,3 +88,27 @@ def test_updating_user(self):
         self.assertEqual(user_instance.email, "test_user@example.com")
         self.assertEqual(user_instance.username, "test_user")
         self.assertEqual(user_instance.password, "test_password")
+
+
+    def test_change_password(self):
+
+        client = APIClient()
```

Comment:
If you're creating a client and separate users, then this should be a separate TestCase and the `client` and the user creation should be in the `setUp` for the new test case.

Reply:
The test cases have been split, one testing if a user can modify their own password and one testing if a user can modify someone else's. 

The client call was removed, since one is inherited and I didn't notice before.